samples/emoji-search/presenter/src/commonMain/kotlin/com/example/redwood/emojisearch/presenter/EmojiSearch.kt

```diff
@@ -75,8 +74,8 @@ fun EmojiSearch(
 ) {
   when (variant) {
     Variant.LAZY_COLUMN -> LazyColumn(httpClient, navigator)
-    Variant.SCROLLABLE_FLEXBOX -> NestedFlexBoxContainers(httpClient, navigator)
-    Variant.BUGGY_COLUMNS -> BuggyNestedColumns(httpClient, navigator)
+    Variant.SCROLLABLE_FLEXBOX -> NestedFlexBoxContainers(httpClient)
+    Variant.BUGGY_COLUMNS -> BuggyNestedColumns()
```

Comment:
moved these two to their own files

I was working on this file (EmojiSearch.kt) and realized there are too many things going on and I got distracted 😥